client/src/components/Ratings.jsx

```diff
@@ -1,75 +1,48 @@
-/* eslint-disable no-unused-vars */
-/* eslint-disable indent */
-
 import React from 'react';
+import { connect } from 'react-redux';
+import _ from 'underscore';
 import ReviewsList from './ReviewsList';
-// Don't have to import the server/index.js?
-
-
-
-// bring all reviews here to have data to go through
-
-// functions needed
-  // componentDidMount? Or alternative?
-
-  // calculate review to the nearest half star
-    // IMPORT Andrew's star function
-
-
-// calculate percentage of reviews recommend product
-// function getPercentageRecommended(results) {
-  // create new array of only 'true' values from results
-  // const trues = results.filter(
-  //   (resultItem.recommend) => (resultItem.recommend === true)
-  //     );
-
-  // return the Length of array of trues divided by original array length
-  // return Math.ceil(trues.length / ratingsArray.length) * 100;
-// }
 
+function getPercentageRecommended(results) {
+  const copy = results.map((result) => result.recommend);
+  const trues = copy.filter(
+    (resultItem) => (resultItem === true)
+  );
+  return Math.ceil(trues.length / results.length) * 100;
+}
```

Comment:
this function will always return 100, use something like this:
```javascript
function getPercentageRecommended(results) {
  const recommended = results.filter((item) => (item.recommend));
  return Math.round(100 * recommended.length / results.length);
}
```

Reply:
Mine was missing a parentheses. Good catch. Fixing for now, can refactor later.